ScoutSuite/providers/aws/rules/findings/cloudformation-stack-with-role.json

```diff
@@ -1,5 +1,5 @@
 {
-    "description": "Role passed to stack",
+    "description": "Stack should not have a role",
```

Comment:
I'd leave it as it was. It's not so much that passing a role to a stack is bad (it's not), but that the user should be knowledgeable of it, and ensure the IAM privileges given to that role follow the principle of least privilege.